app/Helpers/database/static.php

```diff
@@ -39,6 +40,44 @@ function static_addnewregistereduser(string $user): void
     }
 }
 
+function static_addnewhardcoremastery(int $gameId, string $username): void
+{
+    $foundUser = User::firstWhere('User', $username);
+    if ($foundUser->Untracked) {
+        return;
+    }
+
+    $query = <<<SQL
```

Comment:
Using this syntax is inconsistent with the rest of the file (and largely the rest of the codebase). The only other "<<<SQL" is in another recently added function related to the Beaten feature. Does it provide some benefit over a multi-line string? It's not like it's avoiding escaping any characters because the characters that would need to be escaped are eliminated through the bound variables.

Personally, I don't like it because I search for "<<<" when trying to resolve conflicts.

Reply:
The only benefit is improved syntax highlighting for the SQL within the string in VSCode. I'm probably in the minority when it comes to using VSCode, and there's likely more value in removing it for the reasons you've described. Refactored in https://github.com/RetroAchievements/RAWeb/pull/1780/commits/831e813c110827fc54f9e5028e1769ada8564e5f.